packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -101,7 +101,7 @@ export class DynamoDBModelTransformer extends Transformer {
                 ".*" + def.name.value + "Model",
                 ".*" + def.name.value + "DataSource",
                 ".*" + def.name.value + "IAMRole",
-                "^" + def.name.value + "Table",
+                ".*" + def.name.value + "Table",
```

Comment:
Does this need to change?

Reply:
In order to avoid breaking current stacks, the name of the output is `GetAtt${tableLogicalID}Name`, see line 144. Exist another way to map the output to the stack? Are you suggesting adding another **Stack Mapping** and avoid editing the one you pointed? Something like `"GetAtt" + def.name.value + "Table",`.

Reply:
Yeah, I would go with the more specific set so its more clear what is going where.

Reply:
Ok, thanks for the suggestion.